CONTRIBUTING.md

```diff
@@ -402,6 +415,8 @@ This more for situations where "this is broken and here's a fix".
 - After your PR to `master` is approved and merged, merge the most up-to-date `staging` branch into your `hotfix` branch.
 - File a second PR for `hotfix` but this time with [`staging` as the base branch](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request#changing-the-branch-range-and-destination-repository).
 
+<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/4f76c140d109afc6026dc41f8a5af73e88bf0450/components/pr-diagrams/hotfix-pr.png" width=600>
+
```

Comment:
Can we also add a brief summary (a sentence or two) of what's going on in the hotfix diagram here?